app/views/govuk_publishing_components/components/_checkboxes.html.erb

```diff
@@ -34,30 +34,34 @@
       <%= tag.span error, id: "#{id}-error", class: "govuk-error-message" %>
     <% end %>
 
-    <%= tag.div class: "govuk-checkboxes", data: {
+    <%= tag.ul class: "govuk-checkboxes gem-c-checkboxes__list", data: {
       module: ('checkboxes' if has_conditional),
       nested: ('true' if has_nested),
     } do %>
       <% items.each_with_index do |item, index| %>
-        <%= render 'govuk_publishing_components/components/checkbox', {
-          id: id,
-          name: name,
-          item: item,
-          index: index,
-        } %>
-        <% if item[:conditional] %>
-          <%= tag.div item[:conditional], id: "#{id}-#{index}-conditional-#{index}", class: "govuk-checkboxes__conditional govuk-checkboxes__conditional--hidden" %>
-        <% end %>
+        <%= tag.li class: "gem-c-checkboxes__list-item" do %>
```

Comment:
Annoyingly, you could save yourself some markup and some CSS here by having the checkbox div (`govuk-checkboxes__item`) simply be the list element, but of course you can't because that's part of the component. I think this could be another thing to add to the 'things we could do to make this component slightly better if we have time' list.